hack/apidiff.sh

```diff
@@ -156,7 +156,12 @@ run () {
     out="$1"
     mkdir -p "$out"
     for d in "${targets[@]}"; do
-        apidiff -m -w "${out}/$(output_name "${d}")" "${d}"
+        # cd to the path for modules that are intree but not part of the go workspace
+        # per example staging/src/k8s.io/code-generator/examples
+        (
+            cd "${d}"
```

Comment:
Is $d a relative directory path from kube root, or an import path? Doesn't`cd` require a relative file path, and apidiff require an import path?

Reply:
yeah, I got sidetracked, let me push the change to get all the relative paths

Reply:
see next commit

Reply:
> Is $d a relative directory path from kube root, or an import path?

The targets get sanitized earlier to be relative paths.

Reply:
I replaced the logic to get the modules not present in the golang workspace, see https://github.com/kubernetes/kubernetes/pull/129021#discussion_r1862570903